react/features/filmstrip/components/native/Thumbnail.js

```diff
@@ -234,10 +232,13 @@ function _mapDispatchToProps(dispatch: Function, ownProps): Object {
          * @returns {void}
          */
         _onClick() {
-            const { participant } = ownProps;
+            const { participant, tileView } = ownProps;
 
-            dispatch(
-                pinParticipant(participant.pinned ? null : participant.id));
+            if (tileView) {
+                dispatch(toggleToolboxVisible());
```

Comment:
maybe in tile view (and mobile) double click should pin ?

or there will be no way to pin in the tile view ? I guess that's acceptable

Reply:
oh I see you added a pin button, disregard my comment above

Reply:
No, i don't see any reasons to do that. It would just do something that doesn't have any effect here, but has side effects later when the user exits tile view. What do you think?

Reply:
Again, I didn't see Pawel's answer, need to reload my GitHub window more often :D So yeah, we added a new menu item for that, we hope this will provide a good UX